2021/hardy2022re/openreview/packages.tex

```diff
@@ -0,0 +1,19 @@
+% Import packages here
+\usepackage[utf8]{inputenc} % allow utf-8 input
+\usepackage{hyperref}
+
+\usepackage[]{neurips_2019}
+\usepackage{subfig}
+\usepackage{graphicx}
+
+% to avoid loading the natbib package, add option nonatbib:
+%     \usepackage[nonatbib]{neurips_2019}
+
+\usepackage{url}            % simple URL typesetting
+\usepackage{booktabs}       % professional-quality tables
+\usepackage{amsfonts}       % blackboard math symbols
+\usepackage{nicefrac}       % compact symbols for 1/2, etc.
+\usepackage{microtype}      % microtypography
+
+\usepackage[dvipsnames]{xcolor}
```

Comment:
Please comment this import (`xcolor`) as ReScience style files does not work with this (check Readme).